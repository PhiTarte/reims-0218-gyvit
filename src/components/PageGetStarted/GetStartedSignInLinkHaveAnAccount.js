//TODO CREATE SIGN IN TITLE
import React from "react"
import { NavLink } from "reactstrap"

const GetStartedSignInLinkHaveAnAccount = () => (
  <div
    style={{
      marginTop: "120px",
      width: "300px",
      height: "16px",
      fontSize: "16px",
      fontWeight: "normal",
      fontStyle: "normal",
      letterSpacing: "normal",
      textAlign: "left",
      color: "#3b425d"
    }}
  >
    <p>
      Déjà un compte ?
      <NavLink
        href="#"
        style={{
          display: "inline"
        }}
      >
        Se connecter
      </NavLink>
    </p>
  </div>
)

export default GetStartedSignInLinkHaveAnAccount
