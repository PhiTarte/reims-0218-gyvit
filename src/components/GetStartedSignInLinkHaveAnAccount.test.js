import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import GetStartedSignInLinkHaveAnAccount from "./GetStartedSignInLinkHaveAnAccount"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<GetStartedSignInLinkHaveAnAccount />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders link html", () => {
  const tree = renderer.create(<GetStartedSignInLinkHaveAnAccount />).toJSON()
  expect(tree).toMatchSnapshot()
})
