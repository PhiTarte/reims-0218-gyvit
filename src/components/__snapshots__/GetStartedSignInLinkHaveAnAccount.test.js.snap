// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders link html 1`] = `
<div
  style={
    Object {
      "color": "#3b425d",
      "fontSize": "16px",
      "fontStyle": "normal",
      "fontWeight": "normal",
      "height": "16px",
      "letter-spacing": "normal",
      "margin-top": "120px",
      "text-align": "left",
      "width": "300px",
    }
  }
>
  <p>
    Already have an account ?
    <a
      className="nav-link"
      href="#"
      onClick={[Function]}
      style={
        Object {
          "display": "inline",
        }
      }
    >
      Sign in
    </a>
  </p>
</div>
`;
